pydocusign/client.py

```diff
@@ -399,6 +403,51 @@ def get_envelope_recipients(self, envelopeId):
                       envelopeId=envelopeId)
         return self.get(url)
 
+    def delete_envelope_recipients(self, envelopeId, recipientIds):
+        """DELETE to /accounts/{accountId}/envelopes/{envelopeId}/recipients
+        and return JSON.
+
+        ``recipientIds`` is a list of ``recipientId`` values for recipients
+        which will be deleted from the envelope
+
+        NOTE: only `Signer` type recipients are currently supported
+        """
+        if isinstance(recipientIds, basestring):
+            recipientIds = [recipientIds]
+        if not self.account_url:
+            self.login_information()
+        url = '/accounts/{accountId}/envelopes/{envelopeId}/recipients' \
+              .format(accountId=self.account_id,
+                      envelopeId=envelopeId)
+        data = {
+            'signers': [{'recipientId': x} for x in recipientIds]
+        }
+        return self.delete(url, data=data)
```

Comment:
A simple test would be to check all that formatting:

```python
def test_delete_envelope...(self):
    client = ...
    with patch('DocusignClient.delete') as delete:
        client.delete_enveleop_recipients(...)
```

This way we have coverage of this unit of code. The integration with docusign process is another test strategy that could be done with in-house integration plateform.